packages/amplify-provider-awscloudformation/src/push-resources.ts

```diff
@@ -463,8 +463,7 @@ function validateCfnTemplates(context, resourcesToBeUpdated) {
         validateFile(filePath);
       } catch (err) {
         context.print.error(`Invalid CloudFormation template: ${filePath}`);
-
-        throw err;
+        context.print.error(err.message);
```

Comment:
nit: Can just add in the same line
```suggestion
context.print.error(`Invalid CloudFormation template: ${filePath}\n${err.message}`);
```